### ISSUE TITLE
ConfigCache class fatal error

### ISSUE THREAD

**Issue by h3rj4n** [lennerd/vipx-bot-detect#37]
I'm getting the following error trying to enable the cache options:

`Catchable fatal error: Object of class Symfony\Component\Config\ConfigCache could not be converted to string in /vendor/vipx/bot-detect/Vipx/BotDetect/BotDetector.php on line 92`

It solvable by changing the following (from, to):
```
        if ($cache->isFresh()) {
            return $this->metadatas = require $cache;
        }
```
To
```
        if ($cache->isFresh()) {
            return $this->metadatas = require $cache->getPath();
        }
```

**Comment by lennerd** [comment]
Nice catch. Would you like to make a PR for this?

**Comment by h3rj4n** [comment]
A PR? A patch?

**Comment by lennerd** [comment]
Kind of. PR means Pull Request. It's one of the main features of Github. More information: https://help.github.com/articles/using-pull-requests/

**Comment by lennerd** [comment]
Fixed via a6d09f728e19b211aa804e1e09111ef70374cc6a.
